packages/Panel/src/components/Overlay/Overlay.js

```diff
@@ -1,45 +1,31 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/no-unused-prop-types */
 import React from "react";
 import PropTypes from "prop-types";
-import useI18n from "@paprika/l10n/lib/useI18n";
-import * as sc from "./Overlay.styles";
+import { FocusPropTypes } from "@paprika/helpers";
 
 const propTypes = {
-  /** Will call an onClose handler when clicked on (outside of the Panel) */
-  hasOutsideClick: PropTypes.bool,
+  backdropClassName: PropTypes.string,
```

Comment:
I remember having the discussion about this prop, I think haven't age well,  just a random comment. no action required.